### ISSUE TITLE
Pyinstaller AttributeError: 'module' object has no attribute 'error'

### ISSUE THREAD

**Issue by abhrajeetsarder** [pyinstaller/pyinstaller#4111]
Hello, I am just trying to conver my py file to exe by : `pyinstaller -i "D:\image\pyicon.png" --onefile myfile.py` but every time i am facing the same problem.
Problem :: 
```
5095 INFO: PyInstaller: 3.4
5096 INFO: Python: 2.7.15
5100 INFO: Platform: Windows-10-10.0.17134
5115 INFO: wrote D:\file\myFile.spec
5123 INFO: UPX is not available.
5129 INFO: Extending PYTHONPATH with paths
['D:\\file', 'D:\\file']
5178 INFO: checking Analysis
5462 INFO: checking PYZ
5857 INFO: checking PKG
5878 INFO: Bootloader c:\python27\lib\site-packages\PyInstaller\bootloader\Windows-64bit\run.exe
5919 INFO: checking EXE
5921 INFO: Building EXE because EXE-00.toc is non existent
5923 INFO: Building EXE from EXE-00.toc
6128 INFO: SRCPATH [('D:\\weather\\pyicon.png', None)]
6128 INFO: Updating icons from D:\image\pyicon.png to c:\users\subhra~1\appdata\local\temp\tmpjsupre
Traceback (most recent call last):
  File "c:\python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27\Scripts\pyinstaller.exe\__main__.py", line 9, in <module>

  File "c:\python27\lib\site-packages\PyInstaller\__main__.py", line 111, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "c:\python27\lib\site-packages\PyInstaller\__main__.py", line 63, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "c:\python27\lib\site-packages\PyInstaller\building\build_main.py", line 838, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "c:\python27\lib\site-packages\PyInstaller\building\build_main.py", line 784, in build
    exec(text, spec_namespace)
  File "<string>", line 32, in <module>
  File "c:\python27\lib\site-packages\PyInstaller\building\api.py", line 433, in __init__
    self.__postinit__()
  File "c:\python27\lib\site-packages\PyInstaller\building\datastruct.py", line 158, in __postinit__
    self.assemble()
  File "c:\python27\lib\site-packages\PyInstaller\building\api.py", line 522, in assemble
    icon.CopyIcons(tmpnm, self.icon)
  File "c:\python27\lib\site-packages\PyInstaller\utils\win32\icon.py", line 190, in CopyIcons
    except win32api.error as W32E:
AttributeError: 'module' object has no attribute 'error'
```

Can someone tell me, How can i fix this probelm?

**Comment by bitanga** [comment]
I have the exact same problem, does anyone knows how to fix this? My pyinsaller worked well 2 months ago..

**Comment by aoloe** [comment]
I have the same issue.

It works correctly without the icon but fails with an icon.

I've also commented in https://github.com/pyinstaller/pyinstaller/issues/3944#issuecomment-498394160 with a traceback and a zip with the icons that are failing...

**Comment by BruceChen034020** [comment]
Should this be closed?
#3944 was closed.
This error is raised when the icon is collapsed or in incorrect format.

If someone wants to work on this error, 
this is reproducible when assigning a .jpg file to the icon.

**Comment by BoboTiG** [comment]
The module error was fixed with 0b6f44c789e83a7298128caa0c108dc5b0472300. Therefore I am closing the issue.
